packages/jest-types/src/Config.ts

```diff
@@ -273,6 +274,7 @@ export type InitialOptions = Partial<{
   notifyMode: string;
   onlyChanged: boolean;
   onlyFailures: boolean;
+  openHandlesTimeout: number;
```

Comment:
Is this CLI only option? I think it should be added only to `Argv` type below. Good example could be `runInBand`.

`InitialOptions` is exported as `Config` from `'jest'` package. This type users can use for `jest.config.js` default export.

https://github.com/facebook/jest/blob/68e702526c1489abc669ddb2f6364f880848bd77/packages/jest/src/index.ts#L19

Reply:
Ah.. It can be used in config file too. In this case it would be good to document it in this file too: https://github.com/facebook/jest/blob/main/docs/Configuration.md

And to work was CLI option it should be added here: https://github.com/facebook/jest/blob/68e702526c1489abc669ddb2f6364f880848bd77/packages/jest-cli/src/args.ts#L244-L249